### PR TITLE
Add cloud formation scripts and fix some bugs

### LINKED ISSUE #482: Broker test automation using testgrid
**Description:**
We need to automate ballerina-message-broker tests using WSO2-TestGrid.

### PR DESCRIPTION
**Pull request by hasithajayasundara**

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -19,58 +19,47 @@ The tool allows developers to publish/consume messages to/from queues and topics
...
-throughput= <throughput_need_to_be_maintained>
+JmeterHome = <jmeter_home>
+ThreadCount = <number_of_threads>
+MessageSize = <size_of_the_message>
+NumberOfMessages = <number_of_messages_need_to_be_published>
+Throughput= <throughput_need_to_be_maintained>
+```
+2.Include the following details in broker_test_infrastructure.properties file which is located at resources/.
+```properties
+HostURL=<host_url>
```
Comment (sdkottegoda) [inline tools/README.md]: Can we default the hostURL to 'localhost' if this is left empty?

File: tools/README.md [hunk 2]
```diff
@@ -19,58 +19,47 @@ The tool allows developers to publish/consume messages to/from queues and topics
...
 ```
-2.Run ```./broker_test_publisher.sh -p <location_of_properties_file> -d queue/topic```
 
-- -p is an optional paramter.If not provided ```resources/broker_test_publisher.properties``` is set as the properties file location 
+3.Run ```./broker_test_publisher.sh -i <location_of_infrastructure_properties_file> -t<location_of_testplan_properties_file> -d queue/topic```
+
+- -i and -t is are optional paramter.If not provided ```resources/broker_test_infrastructure.properties``` and ```resources/broker_test_plan.properties``` are set as default file locations
```
Comment (sdkottegoda) [inline tools/README.md]: is are optional paramter. Shall we correct the grammar

File: tools/README.md [hunk 3]
```diff
@@ -19,58 +19,47 @@ The tool allows developers to publish/consume messages to/from queues and topics
 
 ### Publish messages to queues/topics
 
-1. Include the following details in broker_performance_test_publisher.properties file which is located at resources/.
+1.Include the following details in broker_test_plan.properties file which is located at resources/.
```
Comment (sdkottegoda) [inline tools/README.md]: A space needs to be there after the period. Shall we correct in all places?

Comment (hasithajayasundara) [reply tools/README.md]: Done

**Pull request merged**